# HG changeset patch
# User Rik <rik@octave.org>
# Date 1375824038 25200
#      Tue Aug 06 14:20:38 2013 -0700
# Node ID b1fd3dc31c4218f84b35a3657e744f8eade08d02
# Parent  b90915ba489d1a9d0610c3879fd0c60b028c987c
struct2hdl.m: Call image() with a single piece of data to avoid loading default.img.

* scripts/plot/struct2hdl.m: Call image() with a single piece of data to avoid
loading default.img.

diff --git a/scripts/plot/struct2hdl.m b/scripts/plot/struct2hdl.m
--- a/scripts/plot/struct2hdl.m
+++ b/scripts/plot/struct2hdl.m
@@ -300,17 +300,17 @@ function [h, sout] = createpatch (s, par
 endfunction
 
 function h = createtext (s, par)
   h = text ("parent", par);
   addmissingprops (h, s.properties);
 endfunction
 
 function h = createimage (s, par)
-  h = image ("parent", par);
+  h = image (1, "parent", par);
   addmissingprops (h, s.properties);
 endfunction
 
 function h = createsurface (s, par)
   h = surface ("parent", par);
   addmissingprops (h, s.properties);
 endfunction
 
